docs/developer/contributing/development-functional-tests.asciidoc

```diff
@@ -6,7 +6,7 @@ We use functional tests to make sure the {kib} UI works as expected. It replaces
 [discrete]
 === Running functional tests
 
-The `FunctionalTestRunner` is very bare bones and gets most of its functionality from its config file, located at {blob}test/functional/config.js[test/functional/config.js] or {blob}x-pack/test/functional/config.js[x-pack/test/functional/config.js]. If you’re writing a plugin outside the {kib} repo, you will have your own config file.
```

Comment:
[These links are broken on the docs master branch](https://www.elastic.co/guide/en/kibana/master/development-tests.html#_running_functional_tests). Hopefully with this change that gets fixed.

Reply:
Hmm, I doubt it but if someone from @elastic/kibana-docs wants to chime in how to fix them I'm be happy to include a fix

Reply:
The latest docs build (and its link checks) succeeded, and the content LGTM here: https://kibana_130983.docs-preview.app.elstc.co/guide/en/kibana/master/development-tests.html#_running_functional_tests

Reply:
There are other unresolved attributes lower in the file, but I'll investigate those in a separate PR.  Thanks for pointing out the problem, @TinaHeiligers !